fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/DepositAccountTermAndPreClosure.java

```diff
@@ -315,11 +316,7 @@ public boolean isTransferInterestToLinkedAccount() {
     }
 
     public boolean isAfterExpectedFirstDepositDate(final LocalDate compareDate) {
-        boolean isAfterExpectedFirstDepositDate = false;
-        if (this.expectedFirstDepositOnDate != null) {
-            isAfterExpectedFirstDepositDate = compareDate.isAfter(getExpectedFirstDepositOnDate());
-        }
-        return isAfterExpectedFirstDepositDate;
```

Comment:
Arent these changes irrelevant to [FINERACT-1921](https://issues.apache.org/jira/browse/FINERACT-1921)? Please extract them into a separate PR if so!